monitor/monitoring-dam-generator-service/pom.xml

```diff
@@ -1,77 +1,78 @@
 <?xml version="1.0"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>eu.seaclouds-project.monitor</groupId>
-    <artifactId>monitor</artifactId>
-    <version>0.8.0-SNAPSHOT</version>
-  </parent>
-  <artifactId>monitoring-dam-generator-service</artifactId>
-  <name>monitoring-dam-generator-service</name>
-  <url>http://maven.apache.org</url>
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <main.class>eu.seaclouds.monitor.monitoringdamgeneratorservice.ServiceLauncher</main.class>
-  </properties>
-  <dependencies>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>eu.seaclouds-project.monitor</groupId>
+		<artifactId>monitor</artifactId>
+		<version>0.8.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>monitoring-dam-generator-service</artifactId>
+	<name>monitoring-dam-generator-service</name>
+	<url>http://maven.apache.org</url>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<main.class>eu.seaclouds.monitor.monitoringdamgeneratorservice.ServiceLauncher</main.class>
```

Comment:
Do you mean the main.class tag? It is simply recalled in the maven-shade plugin to point to the right main. The class eu.seaclouds.monitor.monitoringdamgeneratorservice.ServiceLauncher is the application starter.